.github/workflows/set-automatic-deploys-enabled.yaml

```diff
@@ -4,12 +4,12 @@ on:
   workflow_dispatch:
     inputs:
       resetImageTag:
-        description: 'Whether to reset image tag to main'
+        description: 'Reset image tag to main'
         required: false
         default: false
         type: boolean
       automaticDeploysEnabled:
-        description: 'Set whether automatic deploys is active or not'
+        description: 'Activate automatic deploys'
```

Comment:
Just to check I've understood the right meaning here: if I trigger the workflow with this set to false, is it meant to leave the setting as is, or is it meant to set it to false?